app/components/DocumentPreview/DocumentPreview.js

```diff
@@ -105,6 +105,7 @@ class DocumentPreview extends React.Component<Props> {
           {document.isTemplate && showTemplate && (
             <Badge primary>Template</Badge>
           )}
+          {document.new && <Badge primary>New</Badge>}
```

Comment:
Hoping to get the different color treatment here, I really need a better system for `Badge`s – adding a prop for `yellow` would probably be fine.